src/lib/linux-api/build.rs

```diff
@@ -1,35 +1,38 @@
-use shadow_build_common::ShadowBuildCommon;
+fn main() {
```

Comment:
Slight preference to keep `main` small by instead have the `cfg` guard separately for the `run_cbindgen` definition and the call-site in `main`

Reply:
I added the `cfg` to the `use`, `run_cbindgen`, and scoped contents of `main`.